### PR TITLE
Feat: added .env.example file, pr template fix README.md

### LINKED ISSUE #29: Create an .env.example template file
This file will guide anyone cloning this repo to reproduce environment variables, replacing the examples with their own.

### PR DESCRIPTION
**Pull request by Rajkumar-justcoder**
Fix : README.md 
Feat: Added .env.example file, pr template 

FIx #29

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+PORT=
+MONGO_URI=
+REDISUSER=
+REDISPASSWORD=
+REDISHOST=
+REDISPORT=
```
Comment (amos-kibet) [inline .env.example:6]: We paused use or Redis, maybe we will get back to it in the future, when we find a good use case for it. We forgot to scrap out code related to Redis, but it's fine.

**Review by amos-kibet — APPROVED** [review]

File: .env.example (hunk 1 above)
Comment (Rajkumar-justcoder) [reply .env.example:6]: U could told me to make changes 
but now pr is merged :(

Comment (amos-kibet) [reply .env.example:6]: That is not a big issue. At some point, those env variables will be needed

Comment (Rajkumar-justcoder) [reply .env.example:6]: Ah yeah got that,
yup that also make sense

**Pull request merged**